src/views/Instance/data.js

```diff
@@ -69,6 +71,14 @@ const _processGraphNodes = (processedData, nodesParentsDict, datasetContent, nod
       selector: `node.${nodesGroupName}:selected`,
       style: { ...getDefaultSelectedNodeStyle(theme), ...nodesGroupMetadata.style },
     });
+    processedData.stylesheet.push({
+      selector: `node:parent`,
+      style: { ...getDefaultCompoundNodeStyle(theme) },
+    });
```

Comment:
Can integrators still define custom styles for compound nodes (e.g. "Bar" entities) in `DATA_CONTENT` in _src/config/InstanceVisualization.js_ ?